### ISSUE TITLE
The Institute of Botany, Chinese Academy of Sciences/Journal of Systematics & Evolution

### ISSUE THREAD

**Issue by mark02139** [CDL-Dryad/dryad-product-roadmap#2361]
**Name to appear on membership page:** 
Institute of Botany, Chinese Academy of Sciences

**Payment plan:**
The Institute of Botany CAS is an existing member, but has just signed a new membership agreement that includes quarterly invoicing for datasets submitted.

**Journals included:** 
Journal of Systematics & Evolution

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
No known change.

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
No known change.

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
n/a

**Email contacts to discuss integration of journal metadata with Dryad:**
n/a

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)

**Comment by ryscher** [comment]
Settings updated.
